lore/species/tajaran/The Great Frost.md

```diff
@@ -0,0 +1,85 @@
+
+Finally, as food and material scarcity no longer became the primary fear of the people attention now turned to the dangers of the frost land. The internal threat of the Shards of Messa and the external threats of dangerous predators became the primary concern. After the Great Frost began the vicious and durable predators from the poles began to migrate, spared in the first few decades due to distance the tajaran began to see increased presence of the deadliest predators in the wastelands. The most common of which was the Savik, vicious and brutal predators with a temper and viciousness to match their strength. Rarer among them were the White Renders, sometimes referred to as Messa�s Claws, huge hulking beasts long of limb and able to outrun even the fastest tajaran. Battles between these apex predators and the Sun Wardens became the most common cause of death. Holds without well constructed defenses also fell prey to attacks from these beasts searching for an easy meal. To combat this problem a new weapon was created, one which origins are unknown, yet many attribute its design to Akari Tal�Wrack himself.
+
+The Glare of S�randarr, the first proto-energy weapon developed by the tajaran people. Before its creation many tajarans relied on melee weapons and the lucky ones kinetic rifles and pistols. The Glare of S�randarr entirely changed the field of combat. The apex predators of the frost lands were armored and durable, able to take many blows and bullets before succumbing to their wounds. However, the pure heat and intensity of this newly created weapon proved not only effective but immensely lethal to these creatures. Among the Sun Wardens and hold guard forces these weapons were commonly referred to in slang as slag cannons for their curious design. �Slag cannons� came in both pistol and rifle designs, powered by special battery-like devices charged from the hearth engines. The weapons, when turned on, heated themselves like portable furnaces from a central core that super heated metal pits in a heavy magazine. When the trigger was pulled a primed hammer would slam into the core, the resulting miniature explosion sending the super heated pits out so fast they appeared as a beam of energy. A single well placed shot could down a rampaging savik, while two or three would bring down a white render.

```

Comment:
"Before its creation, many tajarans relied on melee weapons—or kinetic rifles and pistols, if they were lucky."
"... core that superheated metal pits..."
"When the trigger was pulled, a primed..."